components/footer/index.tsx

```
@@ -41,151 +49,319 @@ export function Footer() {
+            *{t("DISCLAIMER")}
+          </div>
+          
+          <div className="flex items-center gap-4">
+            <ThemeToggler compact />
+            
+            {/* Enhanced scroll to top button */}
+            <button
+              onClick={scrollToTop}
+              className="group relative p-3 rounded-2xl bg-gradient-to-br from-white/50 to-white/30 dark:from-gray-800/50 dark:to-gray-900/30 backdrop-blur-lg border border-white/30 dark:border-gray-700/40 hover:border-blue-300/50 dark:hover:border-blue-500/30 transition-all duration-500 hover:shadow-xl hover:shadow-blue-500/10 hover:scale-110 hover:-translate-y-1"
+              aria-label="Scroll to top"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)